syntax = "proto3";


service CameraService {
 
    rpc SendCameraData (CameraDataRequest) returns (CameraDataResponse);
}


message CameraDataRequest {
 
   bytes image = 1;
 
   string timestamp = 2;
   
   int32 frameNumber = 3;
}


 message CameraDataResponse {

   string acknowledgment = 1;
}
